code-challenges.test.js

```diff
@@ -18,39 +18,166 @@
+//
+// ● fibonacci › Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence
 
 // b) Create the function that makes the test pass.
 
+fibonacci = (number)=>{
+    return new Array(number).fill(0).map((value,index, currentArray)=>{
+        if (value > 2 ){
+          return "pick a number greater than 2, yo"
+        }else if (index === 0){
+            currentArray[index] = 1;
+        }else if(index === 1){
+            currentArray[index] = 1;
+        }else{
+            currentArray[index] = currentArray[index-2]+currentArray[index-1];
+        }
+        return currentArray[index];
+    });
+}
+//
+// console.log(fibonacci(6))
+// console.log(fibonacci(10))
+
+// PASS  ./code-challenges.test.js
+//  fibonacci
+//    ✓ Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence (1 ms)
+
+
+
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+// const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 // Expected output: [-9, 7, 9, 199]
 
-const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+// const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
 
+// psuedo code
+// create a function
+// type of to find number
+//  .filter iteration for odd numbers typeof
+// sort odd numbers from least to greatest by comparing if value
+
+// const numArry = fullArr1.filter(
+//   value => typeof value === 'number'
+// )
+//
+// const numOdds = numArry.filter(number => {
+//   return number % 2 !==0
+// })
+//
+// const numSort = numOdds.sort((a, b) => a - b)
+
+describe("finishedArray", () => {
+  it("Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest", () => {
+    const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+    const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+    expect(finishedArray(fullArr1)).toEqual([-9, 7, 9, 199])
+    expect(finishedArray(fullArr2)).toEqual([-823, 7, 23])
+  })
+})
+
+// FAIL  ./code-challenges.test.js
+//  finishedArray
+//    ✕ Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest
+//
+//  ● finishedArray › Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest
+
 // b) Create the function that makes the test pass.
 
+finishedArray = (n) => {
```

Comment:
Woohoo! Mission accomplished. We will keep going. If you are looking for a further challenge...Look over the function for a refactor. 

Let's make a one-liner return----->Since you have used filter twice, let's use the && to connect the evaluations. Then chain that sort method at the end. You will love how it looks at the end.